execute/exectypes/observation.go

```diff
@@ -74,6 +89,10 @@ type Observation struct {
 	// execute report.
 	Messages MessageObservations `json:"messages"`
 	Hashes   MessageHashes       `json:"messageHashes"`
+	// EncodedMsgAndTokenDataSizes are determined during the Get Messages phase
+	// It contains the sum of encoded sizes of messages and their token data counterpart that were observed in same
+	// phase.
+	MessageAndTokenDataEncodedSizes EncodedMsgAndTokenDataSizes `json:"messageEncodedSizes"`
```

Comment:
This doesn't seem like it needs to be part of the observation.

Reply:
Good point 👍